DesignSafe-CI/portal
# enhance: WA-325 button "Unavailable" → "Coming Soon"

## Overview: ##

Change disabled app version button text from "Unavailable" to "Coming Soon".

## PR Status: ##


> [!IMPORTANT]
> After deploy, remove [snippet #19](https://designsafe-ci.org/admin/djangocms_snippet/snippet/19/change/).

## Related Jira tickets: ##

* [WA-325](https://tacc-main.atlassian.net/browse/WA-325)

## Summary of Changes: ##

* **changed** button text

## Testing Steps: ##
1. Replicate [an app version list with a disabled app](https://designsafe-ci.org/use-designsafe/tools-applications/simulation/ee-uq/).
2. Verify disabled app variant button reads "Coming Soon".

> [!NOTE]
> Production website is using a snippet to change the text.

## UI Photos:
Skipped.

## Notes: ##

[Text change confirmed](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1724349856897879?thread_ts=1724348796.204269&cid=CQUA4D5KJ)
